k8s/prod/common/bsp/bulk-scan-orchestrator.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: bulk-scan-orchestrator
+  namespace: bsp
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: bulk-scan-orchestrator
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: bulk-scan-orchestrator
+    version: 0.2.0
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/bulk-scan/orchestrator:latest
+      environment:
+        CCD_LOG_LEVEL: none
+        IDAM_API_URL: "https://idam-api.platform.hmcts.net"
+        IDAM_CLIENT_REDIRECT_URI: "https://rpe-bulk-scan-processor-prod.service.core-compute-prod.internal/oauth2/callback"
```

Comment:
I wonder why this url needed. The uri is not valid - host doesn't exist and even if it did - endpoint doesn't. My guess it is present because IDAM requires it but no use 🤷‍♂. Just wondering here

Reply:
not needed, removed